lib/shopify_api/bulk/query.ex

```diff
@@ -75,6 +75,24 @@ defmodule ShopifyAPI.Bulk.Query do
     end
   end
 
+  @spec fetch_url!(AuthToken.t(), String.t()) :: String.t()
+  def fetch_url!(%AuthToken{} = token, id) do
+    query = """
+      query {
+        node(id: "#{id}") {
+          ... on BulkOperation {
+            url
```

Comment:
we don't want to keep [the](https://shopify.dev/api/usage/bulk-operations/queries#download-result-data) `partialDataUrl`?

Reply:
Is there a reason to?

Reply:
From [the docs](https://shopify.dev/api/usage/bulk-operations/queries#download-result-data) it seems like if any failure occurs with the operation some data can be still available at `partialDataUrl`. But we probably want to handle that failure instead of working with the partial data?

Reply:
The use case we are building this for can't use partial data, and trying to support that will add complexity to this function's interface, so I'm inclined to leave it off, for now.

Reply:
Make sense